src/gui/wizard/webviewpage.h

```diff
@@ -0,0 +1,54 @@
+#ifndef WEBVIEWPAGE_H
+#define WEBVIEWPAGE_H
+
+#include <QWizard>
+#include <QWebEngineView>
+#include <QWebEngineProfile>
+#include <QWebEnginePage>
+#include <QWebEngineUrlRequestInterceptor>
+
+#include "wizard/abstractcredswizardpage.h"
+#include "wizard/owncloudwizardcommon.h"
+#include "wizard/owncloudwizard.h"
+
+#include "ui_webviewpage.h"
+
+namespace OCC {
+
+class WebViewPageUrlRequestInterceptor : public QWebEngineUrlRequestInterceptor
+{
+    Q_OBJECT
+public:
+    WebViewPageUrlRequestInterceptor(QObject *parent = 0);
+    void interceptRequest(QWebEngineUrlRequestInfo &info);
+};
+
+class WebViewPage : public AbstractCredentialsWizardPage
+{
+    Q_OBJECT
+public:
+    WebViewPage(QWidget *parent = 0);
+
+    void initializePage() Q_DECL_OVERRIDE;
```

Comment:
use override in new code, Q_DECL_OVERRIDE is a beast and should die.

Reply:
will take care